cli/lib/cypress.js

```diff
@@ -50,6 +51,22 @@ const cypressModuleApi = {
       })
     })
   },
+
+  cli: {
+    /**
+     * Parses CLI arguments into an object that you can pass to "cypress.run"
```

Comment:
@bahmutov you have 3 different versions of this comment floating around, one in type declarations, one here, and one in cli.js. plz deduplicate. 

maybe a good approach would be to only document it fully in the user-exported type declarations, and then have a single-line comment in the internal code (if you feel it's necessary to explain the function again)

Reply:
nah, I will rather have more comments everywhere than fewer comments, especially since the meaning of individual functions can drift apart